source/Calamari.Common/CalamariFlavourProgramAsync.cs

```diff
@@ -31,6 +31,8 @@ namespace Calamari.Common
 {
     public abstract class CalamariFlavourProgramAsync
     {
+        internal static readonly TimeSpan DefaultRegexMatchTimeout = TimeSpan.FromSeconds(1);
```

Comment:
I think they went with 10 seconds in the Server implementation of this.

Reply:
In Server it is 1 second: https://github.com/OctopusDeploy/OctopusDeploy/blob/3804cabaaf0e162a9b898b00c26e157207174fa2/source/Octopus.Server/Configuration/AppDomainConfiguration.cs#L8

The 10 seconds in PR description was to show an example of manually overriding the default :)

Reply:
Nice, thanks.